### PR TITLE
Small additions to the mindmap

### PR DESCRIPTION
**Pull request by winterrocks**
Added a couple of training courses (LFC 191 and LFC 194) and a sub-heading "Compliance Considerations" because the other branches have the same level headings and now the map has more consistent look. Let's discuss if the heading name is o.k. or if that needs to be changed.

### CODE REVIEW

File: ospo-mindmap/Content/ospomindmap.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ development team of the Linux Project
 
 ### 🔍 Oversee Open Source Compliance
 
+#### Compliance Considerations
```
Comment (anajsana) [inline ospo-mindmap/Content/ospomindmap.md:82]: What other categories apart from "compliance considerations" can fit in the "oversee os compliance section"?

I just want to know if there are more sub-categories we can add apart from "compliance considerations" and list them here too.

Comment (winterrocks) [reply ospo-mindmap/Content/ospomindmap.md:82]: We have "Manage Open Source IT Infrastructure" that includes the "Compliance tooling", but in some organizations the OSPO is the one who is also running scans (SCA) and takes care of this kind of operations, so maybe that could be under "Oversee Open Source Compliance" too?

Comment (anajsana) [reply ospo-mindmap/Content/ospomindmap.md:82]: I've just realized that you added "Compliance Considerations" to replace the missing title from the mind map 👍 

Screenshot 2022-06-16 at 08-31-10 OSPO Mind Map

This segmentation works for me. Also, I prefer the idea of keeping the compliance tooling under the IT infrastructure. IMHO, A single team or OSPO role does no necessarily need to be addressed just on OSPO mindmap responsibility, some OSPOs might be performing several activities (e.g overseeing OS compliance and managing IT infrastructure)

Anyway, we can discuss it with the community in the upcoming AMER & APAC WDA or as a topic for TODO EU Sync and create a new PR if necessary 😉  

Thanks @winterrocks !

Comment (winterrocks) [reply ospo-mindmap/Content/ospomindmap.md:82]: Yep, I saw the lonely green node dot and figured that a heading would be nice for that and came up with "Compliance Considerations", which of course we should discuss and see if there are other and better names.

**Pull request merged**